lib/liquid/block.rb

```diff
@@ -7,6 +7,7 @@ class Block < Tag
     def initialize(tag_name, markup, options)
       super
       @blank = true
+      @body = nil
```

Comment:
Why not `BlockBody.new`? Would avoid the `&.` below.

Reply:
I know it would save the `&.` below but it also initializes the class when it might not be needed. I'm not sure which one is worse.

Reply:
Right. I'm OK to keep it as is.